query.css

```diff
@@ -0,0 +1,208 @@
+@media only screen and (max-width: 400px) {
+  .menu-links {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 1400px) {
+  #line {
+    display: none;
+  }
```

Comment:
- [ ] The standard for Use media queries for the Desktop breakpoint in the CSS file: (the value for min-width property should be in range of 992px - 1100px) 😉

- [ ] Once you are able to fix the queries you can notice a change on the navbar as you intended it to be. 

![Screen Shot 2021-11-05 at 17 10 12](https://user-images.githubusercontent.com/67211919/140588424-20f81f77-8163-49b6-b493-93215088f60d.png)